lib/directory-tree.js

```diff
@@ -37,6 +37,12 @@ function directoryTree (path, options, onEachFile) {
 			options.extensions.indexOf(ext) === -1)
 			return null;
 
+		
+		// Skip file if it matches the exclude regex
```

Comment:
Sorry I meant the place "more towards the beginning of the function". Point was to only write that `if` once, no point in having it in two places.